### ISSUE TITLE
Reduction for the Cross Entropy

### ISSUE THREAD

**Issue by inspirit** [vandit15/Class-balanced-loss-pytorch#1]
thank you for sharing Pytorch implementation,
I was wondering if we need to specify "reduction="none"" for the "binary_cross_entropy_with_logits" function here since we are going to do weighting and reduction on our own later?
https://github.com/vandit15/Class-balanced-loss-pytorch/blob/fb634a6e42ecaa2e3d7e974cd124180c679da6c7/class_balanced_loss.py#L35

**Comment by vandit15** [comment]
Thanks for pointing out. Made necessary changes
